pkgs/top-level/python-packages.nix

```diff
@@ -16265,6 +16265,32 @@ in modules // {
     };
   });
 
+  pyelftools = buildPythonPackage rec {
+    pname = "pyelftools";
+    version = "0.23";
+    name = "${pname}-${version}";
+
+    src = pkgs.fetchurl {
+      url = "https://pypi.python.org/packages/source/p/${pname}/${name}.tar.gz";
+      sha256 = "1pi1mdzfffgl5qcz0prsa7hlbriycy7mgagi0fdrp3vf17fslmzw";
+    };
+
+    checkPhase = ''
+      python test/all_tests.py
```

Comment:
`python` cannot be used because this will break with Python 3.x, in which case the interpreter is `python3`.
Use `${python.interpreter}` instead.